charterafrica-dev.yml

```diff
@@ -0,0 +1,19 @@
+version: "3.8"
+services:
+  charterafrica:
```

Comment:
charterafrica should require mongodb service. I don't see any mongodb service.

Reply:
Since we are going to have database as a separate service, I thought we'd only provide it as a URL

Reply:
No, docker compose needs to connect to local mongo. It won't connect to remove (PROD or STAGING) databases.